.docker/Dockerfile.prod.api

```diff
@@ -41,8 +43,17 @@ COPY --chown=python:python  .docker/etc/uwsgi.ini              .
 COPY --chown=python:python  .docker/etc/wsgi.py                .
 COPY --chown=python:python  apps                               ./apps
 COPY --chown=python:python  .docker/etc/settings_local.py      ./apps/mainsite/
+COPY --chown=python:python  entrypoint.sh                      .
+
+RUN chmod +x entrypoint.sh
+
+# cron requires root permissions
+RUN chmod u+s /usr/sbin/cron 
 
 USER 999
 
 ENV PATH="/badgr_server/venv/bin:$PATH"
-CMD ["uwsgi","--socket", "sock/app.sock", "--ini", "uwsgi.ini"]
\ No newline at end of file
+
+ENTRYPOINT ["./entrypoint.sh"]
+
+# CMD ["uwsgi","--socket", "sock/app.sock", "--ini", "uwsgi.ini"]
```

Comment:
If I understand it correctly, this is now done via the `entrypoint.sh`? Then we also don't need it as a comment